### ISSUE TITLE
LT3 - limits analytically

### ISSUE THREAD

**Issue by AbbyANoble** [StevenClontz/cal1-tbil-2021-checkit#1]
The "nice" (plug and chug) limits always look like polynomial/polynomial. Leads to assumption that the ones with roots or complex fractions are ALWAYS 0/0 situation. Would like to see some of the straightforward plug in chug limits more varied, ones that do not look so nice at first glance. I ask students to explain but I can see formulaic responses showing up, assuming you MUST use a conjugate when there is a square root because that's all they are seeing in their practice.

<img width="203" alt="Screen Shot 2021-09-09 at 5 31 03 PM" src="https://user-images.githubusercontent.com/85640990/132765344-ea4d90d5-4b52-43b6-89e9-5e3b58b0a2ad.png">

**Comment by StevenClontz** [comment]
Yeah, so I'm guessing we need four examples per problem: two poly/poly, one frac/linear (or its reciprocal), and one root/linear (or its reciprocal). Then exactly one of the poly/poly is defined and continuous at the limit, and exactly one of the other two is as well.

**Comment by StevenClontz** [comment]
this isn't broken, so I'm inclined to leave for now. But we'll fix it upstream at https://github.com/StevenClontz/checkit-calculus/issues/4 and get that added here down the road.

**Comment by AbbyANoble** [comment]
That solution works for me. Or just one poly/poly, one root/linear, one frac/linear. Then one of those happens to be plug and chug each time, but it's random which one. But I require 2 passing assessments, so it's likely that they will see different situations. Those with only one passing requirement may want a more robust assessment. I prefer short and sweet. :)

Leaving for now is okay for me. But I may edit my own assessments to account for it.

**Comment by StevenClontz** [comment]
My experience with requiring two passes and only partially covering things resulted in students re-rolling for the seeds they preferred, so my inclination is to make the exercise comprehensive.

**Comment by AbbyANoble** [comment]
Yeah. I get that. But your solution could also lead to re-rolling for the root/linear and fraction/linear. Why do you think it's necessary to have two separate poly/poly (one "nice" and one needing factoring & canceling) but not two for the other situations? (Genuine question, tone not always obvious with typing.)

**Comment by StevenClontz** [comment]
It's a good point - I suppose a question would arguably need six parts to be comprehensive. I guess I feel that the work for root and fraction cases is similar enough (do some algebra *besides* factoring in order to cancel), but I guess someone might prefer the fractions vs finding a conjugate.

**Comment by fragandi** [comment]
We worked on this today and I feel like what is currently in the bank is ok. @AbbyANoble if you are happy about the edits, close this issue.
